package.json

```diff
@@ -15,15 +15,15 @@
   },
   "license": "MIT",
   "dependencies": {
-    "basic-auth-parser": "~0.0.2",
-    "bluebird": "^2.9.30",
-    "middleware-handler": "~0.2.0"
+    "basic-auth-parser": "0.0.2",
+    "bluebird": "^2.11.0",
```

Comment:
It'd be nice if you're able to switch to bluebird 3, I don't know if the breaking changes affect this repo or not though

Reply:
3.x appears to break, at the very least, the tests. Anecdotally, I do think the functionality still works from some earlier testing (I copied the .coffee into another project which is using 3.x)

Reply:
Ah, you need to change
```coffee
request = Promise.promisify(require('request'))
```
to
```coffee
request = Promise.promisify(require('request'), multiArgs: true)
```
and the tests should work again.  I think it would be good to do the bluebird 3 bump as part of this as it also requires a major bump